lld/test/ELF/aarch64-thunk-bti.s

```diff
@@ -0,0 +1,513 @@
+// REQUIRES: aarch64
+// RUN: split-file %s %t
+// RUN: llvm-mc -filetype=obj -triple=aarch64 %t/asm -o %t.o
+// RUN: ld.lld --shared --script=%t/lds %t.o -o %t.so --defsym absolute=0xf0000000
+// RUN: llvm-objdump -d --no-show-raw-insn --print-imm-hex --start-address=0x10000000 --stop-address=0x10000028 %t.so | FileCheck --check-prefix=CHECK-SO %s
```

Comment:
#109553 might be useful to allow removing some llvm-objdump commands, making maintenance more convenient.

`llvm/tools/llvm-objdump/llvm-objdump.cpp:countSkippableZeroBytes` needs to spend time to skip zeroes, so the performance is not as good as multiple llvm-objdump invocations.

Reply:
I've rewritten the test using OutputSections to need much fewer zeroes.